src/input-manager.js

```diff
@@ -134,9 +140,12 @@ input._listener = async function (ch, key) {
         // TODO: Handle utf8 characters
         this.value = value.slice(0, cursor - 1) + value.slice(cursor)
         cursor -= 1
+        // this.value = graphemes.slice(0, graphemes.length - 1).join('')
+        // cursor -= graphemes[graphemes.length - 1].length
```

Comment:
I'm not merging this yet. Is this TODO resolved?

Reply:
It's not fully resolved, I've been trying, I've solved the input and deletion problem using this library, but I've never been able to solve the problem of displaying characters (such as emoji) in two code units, i.e., when typing other characters after the emoji character, the emoji looks as if it occupies the position of only one character and goes blank, I don't have any clue about this, it seems to be that blessed the library has a limitation.

Reply:
So the pr is just so that normal utf8 characters don't show up as '??' , because that's the problem I'm having with it.